### ISSUE TITLE
[Do not open an issue] Website, twitter and subreddit gone

### ISSUE THREAD

**Issue by Roy-Vzla** [popcorntime/popcorntime#1893]
Do I have to worry about my safety using the app?

Thanks.

**Comment by kiriles90** [comment]
Twitter and facebook are gone for a long time now (lately twitter was just a bot repeating r/popcorntime and facebook just gone). There is no point creating new ones since they will ban them again first chance/complain they get.

The website isnt gone, there are server issues on high traffic hours and you might not be able to connect. Try again later or just download from ci.popcorntime.app or the releases page here in github.

Basically nothing else than the subreddit was affected. There is a new subreddit -> [r/popcorntimeapp](https://www.reddit.com/r/PopCornTimeApp) with an explanation as to what happened/what happens now. The repo and the app have also been updated with the new subreddit with https://github.com/popcorn-official/popcorn-desktop/pull/1892 until the situation clears on what happened to the old one and if this will be permanent or not.
